### PR TITLE
Add metrics.addr argument

### PR DESCRIPTION
**Pull request by M0NsTeRRR**
Add the possibility to bind on localhost or specific IP address

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -9,7 +10,10 @@ import (
 )
 
 func main() {
+	var metricsAddr = flag.String("metrics.addr", ":9233", "address for tplink-plug exporter")
```
Comment (fffonion) [inline main.go:13]:
```suggestion
	var metricsAddr = flag.String("metrics.listen-addr", ":9233", "listen address for tplink-plug exporter")
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ Download from [releases](https://github.com/fffonion/tplink-plug-exporter/releas
...
 ```
 
+### Usage
+Use the -h flag to see full usage:
+
+```
+$ tplink-plug-exporter -h
+Usage of tplink-plug-exporter:
+  -metrics.addr string
+        address for tplink-plug exporter (default ":9233")
```
Comment (fffonion) [inline README.md:20]:
```suggestion
  -metrics.listen-addr string
        listen address for tplink-plug exporter (default ":9233")
```

**Comment by fffonion** [comment]
Thank you @M0NsTeRRR ! Some minor comments for phrasing.

**Comment by M0NsTeRRR** [comment]
@fffonion thanks for your quick response :)

If you can tag a new release to avoid clone & setup.py on server it would be cool

**Comment by fffonion** [comment]
@M0NsTeRRR Yes just did! https://github.com/fffonion/tplink-plug-exporter/releases/tag/v0.4.0

**Pull request merged**